pkg/serverless/logs/logs.go

```diff
@@ -336,16 +322,12 @@ func processMessage(
 			message.stringRecord = createStringRecordForReportLog(message, ecs)
 		}
 		if message.logType == logTypePlatformRuntimeDone {
-			serverlessMetrics.GenerateRuntimeDurationMetric(ecs.StartTime, message.time, message.objectRecord.runtimeDoneItem.status, tags, demux)
+			serverlessMetrics.GenerateRuntimeDurationMetric(ecs.StartTime, message.time, tags, demux)
 			ec.UpdateFromRuntimeDoneLog(message.time)
 			ecs = ec.GetCurrentState()
 		}
 	}
 
-	if message.logType == logTypePlatformLogsDropped {
-		log.Debug("Logs were dropped by the AWS Lambda Logs API")
-	}
```

Comment:
Messages with type `logTypePlatformLogsDropped` were never reaching this point and thus this debug statement was never sent. The call to `shouldProcessLog` on line 292 returns false for these messages because `message.stringRecord` and `message.objectRecord.requestID` are both empty.